src/module.c

```diff
@@ -3613,6 +3613,10 @@ int RM_GetContextFlags(RedisModuleCtx *ctx) {
     if (hasActiveChildProcess()) flags |= REDISMODULE_CTX_FLAGS_ACTIVE_CHILD;
     if (server.in_fork_child) flags |= REDISMODULE_CTX_FLAGS_IS_CHILD;
 
+    /* Empty server.loadmodule_queue means that Redis has already started up. */
+    if (listLength(server.loadmodule_queue) == 0)
+        flags |= REDISMODULE_CTX_FLAGS_DENY_LOADING_ON_STARTUP;
+
```

Comment:
the module API [documentation](https://redis.io/docs/reference/modules/modules-api-ref/) is generated from the comments in this file, so the new field should be documented in the comment above.

also, i don't think i like the name. i this is should be negated and mention what it stands for, and not what it's used for.
i'd call it BEFORE_SERVER_STARTUP, or SERVER_STARTUP_PENDING or alike.

while we're here, and now that sentinel can load modules, i think we need a flag for that too.
so that modules that don't wanna be loaded into sentinel can refuse to be loaded.
i.e. add REDISMODULE_CTX_FLAGS_SENTINEL similar to REDISMODULE_CTX_FLAGS_CLUSTER.

p.s. these are interface changes that need to be mentioned in the PR top comment for the purpose of review / approval and release notes.